teams/wg-async-await.toml

```diff
@@ -0,0 +1,12 @@
+name = "wg-async-await"
+subteam-of = "compiler"
+wg = true
+
+[people]
+leads = ["nikomatsakis", "cramertj"]
+members = ["nikomatsakis", "cramertj"]
+
+[website]
+name = "Async-await implementation working group"
+description = "Implementing async-await functionality"
+repo = "https://rust-lang.github.io/compiler-team/working-groups/async-await/"
```

Comment:
Looks like a rebase bug? This file should probably just not be added.

Reply:
Guess it slipped past. Fixed now.